unyt/tests/test_array_functions.py

```diff
@@ -72,6 +72,32 @@
     np.zeros_like,  # works out of the box (tested)
     np.copy,  # works out of the box (tested)
     np.meshgrid,  # works out of the box (tested)
+    np.transpose,  # works out of the box (tested)
+    np.reshape,  # works out of the box (tested)
+    np.resize,  # works out of the box (tested)
+    np.roll,  # works out of the box (tested)
+    np.rollaxis,  # works out of the box (tested)
+    np.rot90,  # works out of the box (tested)
+    np.expand_dims,  # works out of the box (tested)
+    np.squeeze,  # works out of the box (tested)
+    np.flip,  # works out of the box (tested)
+    np.fliplr,  # works out of the box (tested)
+    np.flipud,  # works out of the box (tested)
+    np.delete,  # works out of the box (tested)
+    np.partition,  # works out of the box (tested)
+    np.broadcast_to,  # works out of the box (tested)
+    np.broadcast_arrays,  # works out of the box (tested)
+    np.split,  # works out of the box (tested)
+    np.array_split,  # works out of the box (tested)
+    np.dsplit,  # works out of the box (tested)
+    np.hsplit,  # works out of the box (tested)
+    np.vsplit,  # works out of the box (tested)
+    np.swapaxes,  # works out of the box (tested)
+    np.moveaxis,  # works out of the box (tested)
+    np.nansum,  # works out of the box (tested)
+    np.product,  # is implemented via np.prod
+    np.std,  # works out of the box (tested)
+    np.nanstd,  # works out of the box (tested)
```

Comment:
It occurs to me it would be nice to write tests for all these. Something to think about for future PRs.

Reply:
argh sorry i misread the rest of this PR